spec/views/records/_form.html.erb_spec.rb

```diff
@@ -6,8 +6,19 @@
   end
 
   let(:resource) do
-    ValidatedAssetRepository.new(Image).new
+    Reviewable.new(ValidatedAssetRepository.new(Image).new)
   end
+
+  let(:saved_resource) do
+    resource.tap do |r|
+      r.save
+    end
+  end
+
+  let(:saved_resource_form) do
+    ImageForm.new(saved_resource)
```

Comment:
How about just stubbing `new_record?` to be false instead of saving?  Is there a technical reason to actually persist?